### ISSUE TITLE
Can't fetch cookies on windows

### ISSUE THREAD

**Issue by mraymond** [bertrandom/chrome-cookies-secure#36]
It was working fine a couple days ago, but I tried again recently and it's returning no cookies for any domain on Windows. I tried the google example as well after updating the profile.

Some version info:
Google Chrome | 96.0.4664.93 (Official Build) (64-bit) (cohort: Stable)
Revision | 17531e0a70b4f8108f2418e8b5117f465077710b-refs/branch-heads/4664@{#1229}
OS | Windows 10 Version 21H1 (Build 19043.1348)

And I did notice the Cookies file is empty in the profile I'm checking, in another profile it doesn't exist at all.(I'm assuming it got created when I ran the example)

**Comment by sowhatdoido** [comment]
We were able to solve this problem by patching the following:

```diff
diff --git a/node_modules/chrome-cookies-secure/index.js b/node_modules/chrome-cookies-secure/index.js
index 4d1614d..d188d5b 100644
--- a/node_modules/chrome-cookies-secure/index.js
+++ b/node_modules/chrome-cookies-secure/index.js
@@ -262,7 +262,7 @@ const getCookies = async (uri, format, callback, profile) => {
 	
 	} else if (process.platform === 'win32') {
 
-		path = os.homedir() + `\\AppData\\Local\\Google\\Chrome\\User Data\\${profile}\\Cookies`;
+		path = os.homedir() + `\\AppData\\Local\\Google\\Chrome\\User Data\\${profile}\\Network\\Cookies`;
 
 	} else {
 	
```

It seems like win32 chrome moved their cookie location to Network\Cookies.

I don't know if OSX is going to follow soon, but that might need to be fixed as well.

**Comment by mdpauley** [comment]
I submitted PR #39 for win32

**Comment by rubengmurray** [comment]
I've pushed a new version to npm, `1.3.4 ` which has merged in #39. Please upgrade and let me know on here.

**Comment by rubengmurray** [comment]
https://github.com/bertrandom/chrome-cookies-secure/issues/38#issuecomment-1146320277
